frontend/public/components/graphs/gauge.tsx

```diff
@@ -10,104 +10,106 @@ import { getInstantVectorStats } from './utils';
+  thresholds = DEFAULT_THRESHOLDS,
+  title,
+  usedLabel = 'used',
+
+  // Don't sort, Uses previously declared props
   secondaryTitle = usedLabel,
-  error = false,
 }) => {
   const [ref, width] = useRefWidth();
   const labels = (d) => d.x ? `${d.x} ${usedLabel}` : `${d.y} ${remainderLabel}`;
-  return (
-    <PrometheusGraph className="graph-wrapper--title-center" title={title}>
-      <div ref={ref} className="graph-wrapper--gauge">
-        <PrometheusGraphLink query={query}>
-          <ChartDonutThreshold
-            data={thresholds}
-            height={width} // Changes the scale of the graph, not actual width and height
-            y="value"
-            width={width}
-          >
-            <ChartDonutUtilization
-              labels={labels}
-              data={error ? {y: 0}: data}
-              invert={invert}
-              subTitle={error ? null : secondaryTitle}
-              themeColor={ChartThemeColor.green}
-              thresholds={thresholds}
-              title={error ? 'No Data' : label}
-              theme={theme}
-            />
-          </ChartDonutThreshold>
-        </PrometheusGraphLink>
-      </div>
-    </PrometheusGraph>
-  );
+  return <PrometheusGraph className="graph-wrapper--title-center graph-wrapper--gauge" ref={ref} title={title}>
+    <PrometheusGraphLink query={query}>
+      <ChartDonutThreshold
+        data={thresholds}
+        height={width} // Changes the scale of the graph, not actual width and height
+        y="value"
+        width={width}
+      >
+        <ChartDonutUtilization
+          labels={labels}
+          data={error ? { y: 0 } : data}
+          invert={invert}
+          subTitle={error || loading ? null : secondaryTitle}
+          themeColor={ChartThemeColor.green}
+          thresholds={thresholds}
+          title={loading ? 'Loading' : error ? 'No Data' : label}
+          theme={theme}
+        />
+      </ChartDonutThreshold>
+    </PrometheusGraphLink>
+  </PrometheusGraph>;
 };
 
 export const Gauge: React.FC<GaugeProps> = ({
   humanize = humanizePercentage,
   invert,
   namespace,
   percent = 0,
-  remainderLabel,
   query,
+  remainderLabel,
+  secondaryTitle,
+  theme,
   thresholds,
   title,
   usedLabel,
-  theme,
-  secondaryTitle,
 }) => {
-  const [response, error] = usePrometheusPoll({
+  const [response, error, loading] = usePrometheusPoll({
     endpoint: PrometheusEndpoint.QUERY,
     namespace,
     query,
   });
 
-  const data = response ?
-    getInstantVectorStats(response, null, humanize).map(({label, y}) => ({x: label, y}))[0]
-    : {x: humanize(percent).string, y: percent};
-  return (
-    <GaugeChart
-      query={query}
-      thresholds={thresholds}
-      invert={invert}
-      usedLabel={usedLabel}
-      label={data.x}
-      data={data}
-      remainderLabel={remainderLabel}
-      title={title}
-      theme={theme}
-      error={!!error}
-      secondaryTitle={secondaryTitle}
-    />
+  const [data] = response ? (
```

Comment:
Nit: I think we're normally using the pattern of putting `?` and `:` on the next line. So
```
const [data] = response
  ? getInstantVectorStats(response, null, humanize).map(({ label, y }) => ({ x: label, y }))
  : [{ x: humanize(percent).string, y: percent }];
```